platform-sdk/swirlds-merkledb/src/test/java/com/swirlds/merkledb/MerkleDbDataSourceMetricsTest.java

```diff
@@ -19,9 +19,7 @@
 import static com.swirlds.common.test.fixtures.AssertionUtils.assertEventuallyEquals;
 import static com.swirlds.common.test.fixtures.AssertionUtils.assertEventuallyFalse;
 import static com.swirlds.merkledb.collections.LongListOffHeap.DEFAULT_RESERVED_BUFFER_LENGTH;
-import static com.swirlds.merkledb.test.fixtures.MerkleDbTestUtils.createMetrics;
-import static com.swirlds.merkledb.test.fixtures.MerkleDbTestUtils.getMetric;
-import static com.swirlds.merkledb.test.fixtures.MerkleDbTestUtils.hash;
+import static com.swirlds.merkledb.test.fixtures.MerkleDbTestUtils.*;
```

Comment:
We usually don't use wildcard static imports. I wonder what are spotless rules about it?